programs/budget_program/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-mod budget_processor;
-
-use crate::budget_processor::process_instruction;
+use solana_budget_api::budget_processor::process_instruction;
 
 solana_sdk::process_instruction_entrypoint!(process_instruction);
```

Comment:
getting closer to xmod... ;)

how do you think "process instruction entrypoint" reads when someone wants to use this macro?  granted you imagine no one will, but imagine someone with another plan for packaging...

Reply:
Please rename it as you see fit. I fell into a namespace trap when I tagged it with "entry point".